src/main/java/com/example/cecv_e_commerce/controller/admin/AdminUserController.java

```diff
@@ -0,0 +1,49 @@
+package com.example.cecv_e_commerce.controller.admin;
+
+import com.example.cecv_e_commerce.domain.dto.ApiResponse;
+import com.example.cecv_e_commerce.domain.dto.user.UserDTO;
+import com.example.cecv_e_commerce.domain.dto.user.UserStatusUpdateDTO;
+import com.example.cecv_e_commerce.service.AdminUserService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/admin/users")
+@PreAuthorize("hasRole('ADMIN')")
+public class AdminUserController {
+
+    @Autowired
+    private AdminUserService adminUserService;
+
+    @GetMapping
+    public ResponseEntity<ApiResponse> getAllUsers(
+            @RequestParam(defaultValue = "0") int page,
+            @RequestParam(defaultValue = "10") int size,
+            @RequestParam(defaultValue = "id,asc") String[] sort,
+            @RequestParam(required = false) String search) {
+        Pageable pageable = PageRequest.of(page, size);
+        Page<UserDTO> userPage = adminUserService.getAllUsers(pageable, search, sort);
+        return ResponseEntity.ok(ApiResponse.success("Users retrieved successfully", userPage));
+    }
+
+    @GetMapping("/{userId}")
+    public ResponseEntity<ApiResponse> getUserById(@PathVariable Integer userId) {
```

Comment:
```suggestion
    public ResponseEntity<ApiResponse> show(@PathVariable Integer userId) {
```
ri là đủ nếu tên theo chuẩn restful cho nó gọn